sample-tests-and-e2e-tests/src/use-effect/expensiveOperations.js

```diff
@@ -0,0 +1,4 @@
+export const expensiveOperation1 = () => { console.log("Expensive Operation 1")}
```

Comment:
Do we need have `console.log()` statements here? Don't you think it might pollute the log outputs?

Reply:
I'm not sure what you mean.  These are just stubs of functions which are mocked in the unit tests, so their output doesn't appear anywhere.

Reply:
Sorry, my bad here. I forgot we mocked them.